generators/client/templates/react/src/main/webapp/app/modules/administration/configuration/configuration.tsx.ejs

```diff
@@ -66,8 +66,7 @@ export class ConfigurationPage extends React.Component<IConfigurationPageProps,
 
   reverseIf = (condition, data) => condition ? data.reverse() : data;
 
-  sortPrefix = (configProps, rev) => this.reverseIf(rev, Object.values(configProps)
-                                                            .sort((a, b) => a['prefix'] && a['prefix'].localeCompare(b['prefix'])));
+  sortPrefix = (configProps, rev) => this.reverseIf(rev, Object.values(configProps).sort((a, b) => a['prefix'].localeCompare(b['prefix'])));
```

Comment:
Kinda ugly way to do it, wouldn't mind an advise on how to make it a better way.